src/Parser/EnhancementShaman/Modules/ShamanCore/FuryOfAir.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class FuryOfAir extends Module {
+  furyUptime = 0;
+  maelstromUsed = 0;
+
+  on_initialized() {
+    if (!this.owner.error) {
+      this.active = this.owner.selectedCombatant.hasTalent(SPELLS.FURY_OF_AIR_TALENT.id);
+    }
+  }
+
+  on_byPlayer_applybuff(event) {
+    const spellId = event.ability.guid;
+    console.log("on_byPlayer_applybuff");
+    console.log(event);
+    if (spellId === SPELLS.FURY_OF_AIR.id) {
+      console.log(event);
+      this.maelstromUsed += 3;
```

Comment:
Please define magic numbers outside the class as constants. e.g. `const FURY_OF_AIR_MAELSTROM_COST = 3;`.

Reply:
Updated to use reference value defined in talents